### DISCUSSION TITLE
Impossible teacher verification

### DISCUSSION THREAD

**Discussion by llungster** [community/community#52010] [GitHub Education]
I'm the head coach of a FIRST robotics team and have used GitHub for several years for our team's robot repo. Every so often I do the work to get re-approved and that's fine. This year, nothing works!  I don't have a school badge because I'm a coach that doesn't work at the school during the day so I had the administration write a letter on school letterhead saying that I am employed by the school. According to the instructions, this is supposed to be a valid form of documentation. But it's still not good enough - but something tells me the software capturing the photo doesn't really "read" it. I've updated my profile as instructed but to no avail. My email address is that of the school's domain. What else am I supposed to do? And here's a bit of irony - GitHub is supposed to be a FIRST sponsor.

**Comment by samus-aran** [comment]
Hi @llungster 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Lets see how we can assist ⭐ 

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

2. Often asked to update your profile the application detects differences between your profile information and the evidence you’ve supplied. We do request that students/faculty can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Hope this helps! Let us know how you get along 👍

**Reply by raiarruda** [reply]
Doesn't appear the upload button, only "take picture"

**Reply by samus-aran** [reply]
@raiarruda 👋 

It would be preferable that a student/teacher is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students/teacher submit a live photo of their evidence rather than an upload. We understand that students/teachers may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it.

**Reply by raiarruda** [reply]
I sent my paycheck with my information using obs virtual cam to show my information and didn't work. The document shows my name, my instituition and the date. It is the same kind of document that I send every year.

**Reply by samus-aran** [reply]
I see we have a separate[ discussion here](https://github.com/orgs/community/discussions/53351). Lets continue this conversation on that thread.
